evap/staff/views.py

```diff
@@ -1269,13 +1269,14 @@ def make_questionnaire_edit_forms(request, questionnaire, editable):
 
     if not editable:
         editable_fields = ['visibility', 'is_locked', 'name_de', 'name_en', 'description_de', 'description_en', 'type']
+        editable_question_fields = ['allows_textanswers']
 
         for name, field in form.fields.items():
             if name not in editable_fields:
                 field.disabled = True
         for question_form in formset.forms:
             for name, field in question_form.fields.items():
-                if name != 'id':
+                if name not in editable_question_fields + ['id']:
```

Comment:
It's already disabled in the QuestionForm, why is that necessary here as well?

Reply:
the `allows_textanswers` does not get disabled here